test/Microsoft.TemplateEngine.Orchestrator.RunnableProjects.UnitTests/TemplateConfigTests/FileRenameTests.cs

```diff
@@ -1,9 +1,13 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#nullable enable
```

Comment:
I'm not sure if we should/need-to do this for tests. This complain if something can be null, but sometimes that's exactly what we are trying to test. Getting a `NullReferenceException` in tests doesn't mean we failed, it means that the test worked and caught the problem.

Reply:
We can probably manually disable checks/silence warnings if we want to do that, I think in ideal world 5 years from now, we always write all the code that has this enabled